ws-nextjs-app/cypress/e2e/specialFeatures/atiAnalytics/index.cy.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable import/no-relative-packages */
+import { LIVE_PAGE } from '../../../../../src/app/routes/utils/pageTypes';
+import { assertPageView } from '../../../../../cypress/e2e/specialFeatures/atiAnalytics/assertions';
```

Comment:
in my [PR](https://github.com/bbc/simorgh/pull/12600) i added an alias for the express cypress folder so you could just use `#cypress` here instead of a big long relative import? think this one is closer to being merged so maybe you could make the change here too and ill pull it in to my PR afterwards? all it needs is   ```
 "paths": {
      "#cypress/*": ["../../cypress/*"]
    }
``` added to ` ws-nextjs-app/cypress/tsconfig.json` inside the compiler options.

Reply:
you could probably do similar for the LIVE_PAGE import too if you wanted!

Reply:
actually please ignore me, this only solved the linting error on my machine but it still fails at finding it, think it's a bit tricker than i though - keep the relative paths lol !